react/components/ModalTrigger.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import classnames from 'classnames'
+import { useCssHandles } from 'vtex.css-handles'
+import { useModalDispatch } from './ModalContext'
+
+const CSS_HANDLES = ['triggerContainer']
+
+const ModalTrigger: React.FC<{}> = props => {
+  const { children } = props
+  const handles = useCssHandles(CSS_HANDLES)
+  const containerClasses = classnames(handles.triggerContainer, 'bg-transparent pa0 ma0 outline-0 bw0')
+  const dispatch = useModalDispatch()
+  const handleModalOpen = (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault()
+    e.stopPropagation()
```

Comment:
Why is that?

Reply:
Same as the other one, if I don't do it, might change the page or something depending on where you're using the modal